config/views/courts.py

```diff
@@ -15,12 +15,16 @@ class CourtsTribunalsListView(TemplateViewWithContext):
 
     template_name = "pages/courts_and_tribunals.html"
     page_title = "Judgments and decisions by court or tribunal"
+    page_allow_index = True
```

Comment:
How does this appear in the context as used on line 23 of the middleware? I don't think I get this.

Reply:
This is handled in the  `TemplateViewWithContext` superclass.